build.assets/docker/os.dockerfile

```diff
@@ -44,14 +44,6 @@ RUN set -ex && \
 	systemd-remount-fs.service dev-hugepages.mount sys-fs-fuse-connections.mount \
 	getty.target console-getty.service;
 
-# TODO(r0mant): Disable *iscsi* services cause they may be running on host
-#               In the future we will need to enable them conditionally to
-#               be able to support OpenEBS cStor engine out of the box
-RUN systemctl mask iscsi.service && \
```

Comment:
Can you clarify what would happen now with iscsi service on starting the container? Is it disabled/masked by default?

Reply:
By default the iscsid service will not be started. It will be inactive. It will be started on-demand by the iscsid.socket systemd unit.

```
 systemctl status iscsid.service
● iscsid.service - iSCSI initiator daemon (iscsid)
   Loaded: loaded (/lib/systemd/system/iscsid.service; disabled; vendor preset: enabled)
   Active: inactive (dead)
     Docs: man:iscsid(8)

```

```
systemctl status iscsid.socket
● iscsid.socket - Open-iSCSI iscsid Socket
   Loaded: loaded (/lib/systemd/system/iscsid.socket; enabled; vendor preset: enabled)
   Active: active (listening) since Thu 2020-10-29 14:19:09 UTC; 6min ago
     Docs: man:iscsid(8)
           man:iscsiadm(8)
   Listen: @ISCSIADM_ABSTRACT_NAMESPACE (Stream)
   CGroup: /system.slice/iscsid.socket

```


After I run the test command below it gets activated:

```
iscsiadm -m discovery -t st -p 10.05.02.10

systemctl status iscsid.service
● iscsid.service - iSCSI initiator daemon (iscsid)
   Loaded: loaded (/lib/systemd/system/iscsid.service; disabled; vendor preset: enabled)
   Active: active (running) since Thu 2020-10-29 14:29:41 UTC; 22s ago
     Docs: man:iscsid(8)
  Process: 29004 ExecStartPre=/lib/open-iscsi/startup-checks.sh (code=exited, status=0/SUCCESS)
  Process: 29015 ExecStart=/sbin/iscsid (code=exited, status=0/SUCCESS)
 Main PID: 29019 (iscsid)
    Tasks: 3 (limit: 4915)
   CGroup: /system.slice/iscsid.service
           ├─29018 /sbin/iscsid
           ├─29019 /sbin/iscsid
           └─29020 /sbin/iscsid


```